addons/api/addon/serializers/host-catalog.js

```diff
@@ -4,30 +4,51 @@
  */
 
 import ApplicationSerializer from './application';
+import {
+  TYPE_HOST_CATALOG_PLUGIN_AWS,
+  TYPE_HOST_CATALOG_PLUGIN_AZURE,
+} from '../models/host-catalog';
 
-const fieldsByType = {
-  aws: [
-    'disable_credential_rotation',
-    'region',
+const azureFields = [
+  'disable_credential_rotation',
+  'tenant_id',
+  'client_id',
+  'subscription_id',
+  'secret_id',
+  'secret_value',
+];
+
+const AWSfieldsWithCredentialType = {
+  static: [
```

Comment:
I realize this now means static credential vs dynamic credential but that was not obvious and I was confused as I thought this was a static host catalog, can we make this clearer?

Reply:
I had the same issue. Easy to get confused between static AWS credential and static host catalog.

Reply:
I added a comment, let me know if that helps?

Reply:
Could we call the variable itself something else, maybe staticCredential?

Reply:
updated